### PR TITLE
Fix time start bug in `SnapLogsJournal()`

### PR DESCRIPTION
**Pull request by MonicaisHer**

### CODE REVIEW

File: test/utils/snap.go [hunk 1]
```diff
@@ -82,10 +82,24 @@ func SnapDumpLogs(t *testing.T, name string) {
...
+			time.Sleep(1 * time.Second)
+			logs, _ = Exec(t, fmt.Sprintf(
+				"sudo journalctl --since \"%s\" --no-pager | grep \"%s\"|| true\n\n",
+				start.Format("2006-01-02 15:04:05"),
+				name))
+
+		} else {
+			logIsEmpty = false
+		}
+	}
```
Comment (farshidtz) [inline test/utils/snap.go:101]: The loop and wait doesn't belong here. This function is to get logs and it was doing it just fine (except for the returning exit code 1).

You loop should be in asc tests and call this log function periodically.

**Review by farshidtz — CHANGES_REQUESTED** [review]

File: test/utils/snap.go [hunk 2]
```diff
@@ -83,9 +83,10 @@ func SnapDumpLogs(t *testing.T, name string) {
 
 func SnapLogsJournal(t *testing.T, start time.Time, name string) string {
 	logs, _ := Exec(t, fmt.Sprintf(
-		"sudo journalctl --since \"%s\" --no-pager | grep \"%s\"\n\n",
-		time.Now().Format("2006-01-02 15:04:05"),
+		"sudo journalctl --since \"%s\" --no-pager | grep \"%s\"|| true\n\n",
+		start.Format("2006-01-02 15:04:05"),
```
Comment (farshidtz) [inline test/utils/snap.go:87]: 👀

Comment (MonicaisHer) [reply test/utils/snap.go:87]: Oh... Do I actually need a for loop

Comment (MonicaisHer) [reply test/utils/snap.go:87]: Ok, it works. I removed the loop.

Comment (farshidtz) [reply test/utils/snap.go:87]: Yes this was a bug. 

Maybe on a slow system, 1 second would not be sufficient, but we can address that if it happens.

**Review by farshidtz — APPROVED** [review]

**Pull request merged**